### PR TITLE
HA support using Redis

### PR DESCRIPTION
**Pull request by kjerand**
The integration test will not work because it currently uses localhost instead of a test connection string.

### CODE REVIEW

File: ExampleExtractor/config.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ cognite:
     scopes:
       - ${BF_TEST_SCOPE}
 high-availability:
-  index: 1
-  raw:
-    database-name: ${BF_TEST_DB}
+  index: 0
+  redis:
+    connection-string: "localhost"
     table-name: ${BF_TEST_TABLE}
```
Comment (einarmo) [inline ExampleExtractor/config.yml:18]: I'd prefer if you keep it with raw for now, since that's more example-y.

**Review by einarmo — APPROVED** [review]
Neat! I think you'll have to figure out a way to run Redis in Jenkins. Might be an interesting project for your last few days.

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -10,6 +10,10 @@ jobs:
   build:
     timeout-minutes: 10
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        redis-version: [4, 5, 6]
```
Comment (einarmo) [inline .github/workflows/build-and-test.yml:16]: You don't use node, and this is overkill. Just remove this whole bit and put in a constant in place of `${{ matrix.redis-version }}`

Comment (kjerand) [reply .github/workflows/build-and-test.yml]: Yeah, I was just testing the example code

**Pull request merged**